Documentation/Concepts/Extensions/Index.rst

```diff
@@ -7,9 +7,92 @@
 Extensions
 ==========
 
-.. figure:: /Images/Illustrations/extensions.png
-
 Extensions are pieces of software developed by the TYPO3 community that extend
-the functionality of a TYPO3 installation. Extensions come in many forms
-- from small extensions that carry out specific tasks to larger extensions that
-provide an entire suite of functionality, e.g. the TYPO3 Blog Extension.
+the functionality of a TYPO3 installation. Extensions come in many forms.
+Some are only used for one site and contain mainly the theme of that site.
+These extensions are called a :ref:`site package <creating-a-site-package>`.
+
+..  contents::
+
+..  _concepts-extensions-composer:
+
+Extension vs Composer package
+=============================
+
+If you have worked with other PHP based projects you have probably run across
+`Composer packages <https://dev.to/joemoses33/create-a-composer-package-how-to-29kn>`__.
+
+Each TYPO3 extension is a Composer package of type `typo3-cms-extension`.
+Extensions provided by the TYPO3 Core have type `typo3-cms-framework`.
+
+The minimum needed to define a TYPO3 extension is:
+
+A directory containing a file called :file:`composer.json` with at least
+the following data:
+
+..  code-block:: yaml
+    :caption: packages/my_extension/composer.json
+
+    {
+        "name": "myvendor/my-extension",
+        "type": "typo3-cms-extension",
+        "require": {
+            "typo3/cms-core": "^13.4",
+        },
+        "extra": {
+            "typo3/cms": {
+                "extension-key": "my_extension"
+            }
+        }
+    }
+
+In order to be used the Extension needs to be
+:ref:`installed via Composer <installing-extensions>`.
+
+There is a legacy way to
```

Comment:
Yes, but it is not "unsupported" or "deprecated". We must trad carefully here, and there are also concerns raised from the core team we must not use this terminology "legacy".

Reply:
Let's try to lay opinions aside and look at the facts and what best and most clearly describes it.

Using composer *is* the de facto recommended way to install TYPO3. Indeed, it has clear advantages, for example from a security perspective. Thinking back the last ten years I cannot remember having heard anyone recommend anything else than Composer for TYPO3 installations. To the contrary, I have heard the recommendation of Composer both from a concrete question of best practice and otherwise.

Legacy does not imply that it is officially scheduled for removal, like the case would be if it was deprecated. [Legacy means](https://www.merriam-webster.com/dictionary/legacy#dictionary-entry-2) "of, relating to, or being a previous or outdated computer system" or "of, relating to, associated with, or carried over from an earlier time, technology, business, etc." That is indeed the case here. The non-composer installation methodology and related features have been carried over from an earlier time/technology. 

Classic (like in [Classic Mac OS](https://en.wikipedia.org/wiki/Classic_Mac_OS)) can be a valid synonym, but the word has the disadvantage of also [meaning](https://www.merriam-webster.com/dictionary/classic#dictionary-entry-1) "serving as a standard of excellence," which I think we can agree isn't the meaning we're trying to covey.

Reply:
So legacy *can* mean "carried over from an earlier time" but it can also – you even quoted it first – mean
> of, relating to, or being a previous or outdated computer system

TYPO3 Classic mode is not an **outdated** computer system / application. TYPO3 v13.4.0 installed in classic mode from https://get.typo3.org/version/13 via `wget --content-disposition https://get.typo3.org/13.4.0` is a perfectly valid installation method and not "outdated". Or does get.typo3.org state that?

A term like legacy implies "outdated", this is bad from a marketing perspective, a it causes uncertainty for our users.
Please be aware that there are not just developers, there are users that want plug and play, auto updating hosters and even products like IONOS cloud – see https://www.youtube.com/live/o3BHfJtiQeQ?si=tRqmL1Gd3fc6WrHE&t=22333 – that are built *solely* on TYPO3 classic mode and will only support that.

We need to understand that there is an active part of our community that prefers composer mode over classic mode for good reasons (and that's fine and perfectly valid), but it's not for everyone.

This is a strategic topic. Not something that we as a Core Team or you as Documentation Team should decide. Until it's strategically decided that Classic Mode is to be dropped I urge you to not call it legacy. Please.

…but it's good to discuss that. Thanks @fe-hicking for bringing that up!

Reply:
The discussion while valid does not belong in this PR. The term legacy installation has been established throughout the docuemtation since even before I joined the team. So until there is a descision to rename it once more we will use the term used in TYPO3 explained: https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/Administration/Installation/LegacyInstallation.html